src/components/atoms/Button/Button.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <button :class="classNames" type="button">
+        <slot />
+    </button>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from 'vue';
+
+export default defineComponent({
+    name: 'BaseButton',
+    props: {
+        circular: {
+            type: Boolean,
+            default: false,
+        },
+        flat: {
+            type: Boolean,
+            default: false,
+        },
+    },
+    setup(props) {
+        const classNames = computed(() => {
+            let list = 'font-body';
+
+            if (props.circular) {
+                list += ' rounded-full';
+            } else {
+                list += ' rounded';
+            }
```

Comment:
```suggestion
            let list = 'font-body rounded';

            if (props.circular) {
                list += '-full';
            }
```

Reply:
I understand the proposal but I think it is better to keep the two separate classes to keep it clear. It is two different things with two different effects. Thanks for the proposal anyway !